pkg/proto/nbcontract/v1/clusterconfig.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package nbcontract.v1;
+
+import "pkg/proto/nbcontract/v1/loadbalancerconfig.proto";
+import "pkg/proto/nbcontract/v1/vmtype.proto";
+
+// Cluster Config fields stored in azure.json used by cloud-provider-azure
+message ClusterConfig {
+  string resource_group = 1;
+  string location = 2;
+  VmType vm_type = 3;  
+  string primary_availability_set = 4;
+  string primary_scale_set = 5;
+  ClusterNetworkConfig virtual_network_config = 6;
+  bool use_instance_metadata = 7;
+  LoadBalancerConfig load_balancer_config = 8;
```

Comment:
As you are putting ClusterNetworkConfig definition in this proto. Maybe we should also put LoadBalancerConfig definition this proto too for consistency?

Reply:
moved